src/reformat.c

```diff
@@ -313,6 +333,19 @@ avifResult avifImageRGBToYUV(avifImage * image, const avifRGBImage * rgb)
                             yuvBlock[bI][bJ].y = 0.5f * rgbPixel[1] + 0.25f * (rgbPixel[0] + rgbPixel[2]);
                             yuvBlock[bI][bJ].u = 0.5f * rgbPixel[1] - 0.25f * (rgbPixel[0] + rgbPixel[2]);
                             yuvBlock[bI][bJ].v = 0.5f * (rgbPixel[0] - rgbPixel[2]);
+#if defined(AVIF_ENABLE_EXPERIMENTAL_YCGCO_R)
+                        } else if (state.mode == AVIF_REFORMAT_MODE_YCGCO_RE || state.mode == AVIF_REFORMAT_MODE_YCGCO_RO) {
+                            // Formulas from JVET-U0093.
```

Comment:
I don't have JVET-U0093. I am using ISO/IEC DIS 23091-2:2022(E) (https://www.iso.org/obp/ui/#iso:std:iso-iec:23091:-2:dis:ed-3:v1:en).

Reply:
Update: I found JVET-U0093. I will compare its equations with the equations in ISO/IEC DIS 23091-2:2022(E) tomorrow.

In any case, your code seems to match the equations in ISO/IEC DIS 23091-2:2022(E).